From a8dd26bec6ce2790c421d5973fc0335042875922 Mon Sep 17 00:00:00 2001
From: David Phillips <david@sighup.nz>
Date: Wed, 9 Oct 2019 20:11:43 +1300
Subject: [PATCH] Add missing harfbuzz include dir to search path.

It appears that Arch Linux has either moved harfbuzz headers, or is newly
depending on harbuzz from pango. This patch includes the harfbuzz include dir
in CFLAGS.
---
 config.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/config.mk b/config.mk
index 9b68792..af91126 100644
--- a/config.mk
+++ b/config.mk
@@ -42,6 +42,7 @@ DEPS +=    bat_tray.h \
 CFLAGS +=  -I/usr/include/gtk-3.0 \
            -I/usr/lib/gtk-3.0/include \
            -I/usr/include/pango-1.0 \
+           -I/usr/include/harfbuzz \
            -I/usr/include/cairo \
            -I/usr/include/gdk-pixbuf-2.0 \
            -I/usr/include/atk-1.0 \
-- 
2.23.0

